test/integration/uk/gov/hmrc/exports/repositories/SubmissionRepositorySpec.scala

```diff
@@ -161,12 +161,14 @@ class SubmissionRepositorySpec
       "return all the Submissions" in {
         repo.save(submission).futureValue
         repo.save(submission_2).futureValue
+        repo.save(submission_3).futureValue
 
         val retrievedSubmissions = repo.findAllSubmissionsForEori(eori).futureValue
 
         retrievedSubmissions.size must equal(2)
         retrievedSubmissions must contain(submission)
         retrievedSubmissions must contain(submission_2)
```

Comment:
It's not necessary

Reply:
the line below should check they are contained in the collection